### PR TITLE
Updating with the latest dependency info

### PR DESCRIPTION
**Pull request by nimisha20**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows2016/2047/HighFidelity-Beta-PR11063-f7add1f6c6cd6a443577cf8e7fc93464376a48b1.exe

File: BUILD.md [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+- [OpenSSL](https://www.openssl.org/): Use the latest available version of OpenSSL to avoid security vulnerabilities.
+- [VHACD](https://github.com/virneo/v-hacd)(clone this repository)(Optional)
+
+### CMake External Project Dependencies
+
+These dependencies need not be installed manually. They are automatically downloaded on the platforms where they are required.  
+- [Bullet Physics Engine](https://github.com/bulletphysics/bullet3/releases):  2.83
+- [GLEW](http://glew.sourceforge.net/):   1.13
+- [glm](https://glm.g-truc.net/0.9.8/index.html):  0.9.8
+- [gverb](https://github.com/highfidelity/gverb)
```
Comment (kencooke) [inline BUILD.md]: We no longer use gverb, so this can be removed.

File: BUILD.md [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+- [Oculus SDK](https://developer.oculus.com/downloads/):   1.11 (Win32) / 0.5 (Mac)
+- [OpenVR](https://github.com/ValveSoftware/openvr):   1.0.6 (Win32 only)
+- [Polyvox](http://www.volumesoffun.com/):   0.2.1
+- [QuaZip](https://sourceforge.net/projects/quazip/files/quazip/):   0.7.3
+- [SDL2](https://www.libsdl.org/download-2.0.php):   2.0.3
+- [soxr](https://sourceforge.net/p/soxr/wiki/Home/):   0.1.1
```
Comment (kencooke) [inline BUILD.md]: We no longer use SOXR, so this can also be removed.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/2067/HighFidelity-Beta-PR11063-55e23a31069087ca6d4db8445c0fc72cff3bc050.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/2067/HighFidelity-Beta-PR11063-55e23a31069087ca6d4db8445c0fc72cff3bc050.exe

**Pull request merged**